mod-orders/mod-orders.postman_collection.json

```diff
@@ -5480,6 +5494,268 @@
 							],
 							"_postman_isSubFolder": true
 						},
+						{
+							"name": "Quantity validation",
+							"item": [
+								{
+									"name": "Verify Physical Resource",
+									"event": [
+										{
+											"listen": "prerequest",
+											"script": {
+												"id": "570809ac-8aba-4c7c-b384-8ee934d027c0",
+												"exec": [
+													"let utils = eval(globals.loadUtils);",
```

Comment:
Just a thought... Postman complains `eval can be harmful`. It's possibly complaining because the code is open for injections attacks. I think we can ignore warning because this is a test code. To overcome that warning: is it possible to disable the rule https://github.com/eslint/eslint/issues/4265#issuecomment-151276215 ?